lib/StaticNarrative/exporter/preprocessor.py

```diff
@@ -170,22 +181,27 @@ def _get_data_cell_ref(self, meta: dict, ws_id: int = None) -> str:
         meta = meta["dataCell"]
         ref = None
         if "upas" in meta and ws_id is not None:
-            refs = [deserialize(u, ws_id) for u in meta["upas"].values()]
+            # "upas" may be doubly-nested so check for another "upas" key
```

Comment:
found this situation in some SNs. Not sure where in the narrative this is happening, but somewhere `upas` is being recorded in two different ways, so the code needs to be able to deal with it.